packages/shadydom/src/patch-events.js

```diff
@@ -379,35 +403,14 @@ function shadyDispatchEvent(e) {
   currentTarget = null;
 }
 
-function listenerSettingsEqual(
-  savedListener,
-  node,
-  type,
-  capture,
-  once,
-  passive
-) {
-  let {
-    node: savedNode,
-    type: savedType,
-    capture: savedCapture,
-    once: savedOnce,
-    passive: savedPassive,
-  } = savedListener;
-  return (
-    node === savedNode &&
-    type === savedType &&
-    capture === savedCapture &&
-    once === savedOnce &&
-    passive === savedPassive
-  );
+function listenerSettingsEqual(savedListener, node, type, capture) {
+  let {node: savedNode, type: savedType, capture: savedCapture} = savedListener;
+  return node === savedNode && type === savedType && capture === savedCapture;
```

Comment:
This is probably ok, but ideally I'd want to check what the browser considers equal for options. ie, should these be loose checks?

Reply:
Think these have to exactly match, see https://dom.spec.whatwg.org/#add-an-event-listener.